src/main/java/fact/datacorrection/DrsCalibration.java

```diff
@@ -267,11 +269,20 @@ public Data process(Data data) {
 
     @Override
     public void init(ProcessContext processContext) throws Exception {
-        if (url != null) {
+
+        if (!url.isEmpty()) {
```

Comment:
it seems that in case the property is empty this line is never reached. I assume that streams does not handle empty properties very well.

Reply:
However, searching for a specific sting works `if (!url.equals("a"))` and in the XML `<property name="drsfile" value="a" />`.